### PR TITLE
add two exceptional test cases

### PR DESCRIPTION
**Pull request by qlinsey**
Added test cases to catch exceptional scenarios

### CODE REVIEW

File: cool-core/src/test/java/com/nus/cool/core/model/CoolModelTest.java [hunk 1]
```diff
@@ -15,12 +15,15 @@
 import com.nus.cool.model.CoolLoader;
 import com.nus.cool.model.CoolModel;
 import org.testng.annotations.Test;
+import static org.testng.Assert.assertEquals;
```
Comment (KimballCai) [inline cool-core/src/test/java/com/nus/cool/core/model/CoolModelTest.java:18]: Please just import org.testng.Assert

File: cool-core/src/test/java/com/nus/cool/core/model/CoolModelTest.java [hunk 2]
```diff
@@ -52,6 +55,20 @@ public static void CsvLoaderTest() throws IOException {
         loader.load(cube, schemaFileName, dimFileName, dataFileName, cubeRepo);
     }
```
Comment (KimballCai) [inline cool-core/src/test/java/com/nus/cool/core/model/CoolModelTest.java:57]: we should do the exception test, but you should add such tests for each class.
For example, this CsvFileNonExistTest function should be created in the unit tests for the CsvLoader.
You can check PR #32 to learn how to create a new unit test file for a certain class.

File: cool-core/src/test/java/com/nus/cool/core/model/CoolModelTest.java [hunk 3]
```diff
@@ -62,6 +79,29 @@ public void CubeListTest() throws IOException {
         System.out.println("Applications: " + Arrays.toString(cubes2));
     }
 
+    @Test (priority = 10)
+//show the CUBE list is empty
+    public void CubeEmptyListTest() throws IOException {
+        System.out.println("======================== Cube List Test ========================");
```
Comment (KimballCai) [inline cool-core/src/test/java/com/nus/cool/core/model/CoolModelTest.java:85]: Please update this log.

File: cool-core/src/test/java/com/nus/cool/core/model/CoolModelTest.java [hunk 4]
```diff
@@ -52,6 +55,20 @@ public static void CsvLoaderTest() throws IOException {
         loader.load(cube, schemaFileName, dimFileName, dataFileName, cubeRepo);
     }
 
+    @Test(expectedExceptions = { IOException.class },priority = 10, enabled = false)
```
Comment (KimballCai) [inline cool-core/src/test/java/com/nus/cool/core/model/CoolModelTest.java:58]: We should add logic to check the existence of these files between lines 43-46 in CoolLoader.java

File: cool-core/src/test/java/com/nus/cool/core/model/CoolModelTest.java (hunk 3 above)
Comment (KimballCai) [reply cool-core/src/test/java/com/nus/cool/core/model/CoolModelTest.java:85]: but I do not think that we should add this test function. It only shows a minor difference from the previous one.

**Review by KimballCai — CHANGES_REQUESTED** [review]

File: cool-core/src/test/java/com/nus/cool/core/model/CoolModelTest.java (hunk 3 above)
Comment (qlinsey) [reply cool-core/src/test/java/com/nus/cool/core/model/CoolModelTest.java]: agree,thanks

File: cool-core/src/test/java/com/nus/cool/core/model/CoolModelTest.java (hunk 4 above)
Comment (qlinsey) [reply cool-core/src/test/java/com/nus/cool/core/model/CoolModelTest.java]: added another separate test case for csv loader

**Pull request closed without merging**